shotover-proxy/tests/cassandra_int_tests/collections.rs

```diff
@@ -57,57 +57,148 @@ fn get_type_str(col_type: ColType) -> &'static str {
+            ResultValue::BigInt(1844674407371),
+            ResultValue::BigInt(1844674407372),
+        ],
+        ColType::Blob => vec![
+            ResultValue::Blob(vec![0, 0, 0, 0, 0, 0, 0, 10]),
+            ResultValue::Blob(vec![0, 0, 0, 0, 0, 0, 0, 11]),
+            ResultValue::Blob(vec![0, 0, 0, 0, 0, 0, 0, 12]),
+        ],
+        ColType::Boolean => vec![ResultValue::Boolean(false), ResultValue::Boolean(true)],
+        ColType::Counter => vec![
+            ResultValue::Counter(12),
+            ResultValue::Counter(13),
+            ResultValue::Counter(14),
+        ],
+        ColType::Decimal => vec![
+            ResultValue::Decimal(vec![0, 0, 0, 3, 4, 87]),
+            ResultValue::Decimal(vec![0, 0, 0, 3, 4, 88]),
+            ResultValue::Decimal(vec![0, 0, 0, 3, 4, 89]),
+        ],
+        ColType::Double => vec![
+            ResultValue::Double(1.11.into()),
+            ResultValue::Double(1.12.into()),
+            ResultValue::Double(1.13.into()),
+        ],
+        ColType::Float => vec![
+            ResultValue::Float(1.11.into()),
+            ResultValue::Float(1.12.into()),
+            ResultValue::Float(1.13.into()),
+        ],
+        ColType::Int => vec![
+            ResultValue::Int(1),
+            ResultValue::Int(2),
+            ResultValue::Int(3),
+        ],
+        ColType::Timestamp => vec![
+            ResultValue::Timestamp(1296705660000),
+            ResultValue::Timestamp(1296705720000),
+            ResultValue::Timestamp(1296705780000),
+        ],
+        ColType::Uuid => vec![
+            ResultValue::Uuid(
+                uuid::Uuid::parse_str("84196262-53de-11ec-bf63-0242ac130001").unwrap(),
+            ),
+            ResultValue::Uuid(
+                uuid::Uuid::parse_str("84196262-53de-11ec-bf63-0242ac130002").unwrap(),
+            ),
+            ResultValue::Uuid(
+                uuid::Uuid::parse_str("84196262-53de-11ec-bf63-0242ac130003").unwrap(),
+            ),
+        ],
+        ColType::Varchar => vec![
+            ResultValue::Varchar("varchar 1".into()),
+            ResultValue::Varchar("varchar 2".into()),
+            ResultValue::Varchar("varchar 3".into()),
+        ],
+        ColType::Varint => vec![
+            ResultValue::VarInt(vec![3, 5, 233]),
+            ResultValue::VarInt(vec![3, 5, 234]),
+            ResultValue::VarInt(vec![3, 5, 235]),
+        ],
+        ColType::Timeuuid => vec![
+            ResultValue::TimeUuid(
+                uuid::Uuid::parse_str("84196262-53de-11ec-bf63-0242ac130001").unwrap(),
+            ),
+            ResultValue::TimeUuid(
+                uuid::Uuid::parse_str("84196262-53de-11ec-bf63-0242ac130002").unwrap(),
+            ),
+            ResultValue::TimeUuid(
+                uuid::Uuid::parse_str("84196262-53de-11ec-bf63-0242ac130003").unwrap(),
+            ),
+        ],
+        ColType::Inet => vec![
+            ResultValue::Inet("127.0.0.1".parse().unwrap()),
```

Comment:
Inet is only IPv4?

Reply:
https://docs.datastax.com/en/cql-oss/3.x/cql/cql_reference/cql_data_types_c.html 
![image](https://github.com/shotover/shotover-proxy/assets/5120858/5b6548c8-d8a8-4d72-bf7c-5fb361977bb1)

IPV4 or IPV6